src/pages/CommandView/form-data/get-schema/get-schema.test.ts

```diff
@@ -0,0 +1,1004 @@
+import { Instance, Parameter } from 'types/backend-types'
```

Comment:
This was done via TDD. I.e., what was desired was known, so a test was written before the implementation code. However, there is at least a little bit of functionality left out (for example, parameters with a `regex` field, as well as others), which will be picked up under the "Dynamic Choices" ticket.